packages/integration-tests/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ["@matterlabs/eslint-config-vue"],
```

Comment:
Why `eslint-config-vue`? We don't use vue in integration-tests

Reply:
@Romsters what you can advice to use here?